modular_zubbers/master_files/code/controllers/subsystem/tts.dm

```diff
@@ -0,0 +1,7 @@
+// Note: This is terrible and permanently disables tts without fixing the underlying issue
```

Comment:
We don't have TTS anyway and this can be fixed if we do get TTS. This is a bandaid solution, yes. But this is a button that crashes the server.

Reply:
Fair point